apps/linode-marketplace-influxdb/roles/common/tasks/ufw_rules.yml

```diff
@@ -0,0 +1,28 @@
+---
+# set app specific ufw rules 
+- name: Allow all access to tcp port 80
+  community.general.ufw:
+    rule: allow
+    port: '80'
+    proto: tcp
+
+
```

Comment:
```suggestion
```

Reply:
removed in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)